### PR TITLE
Feature/multi search func

### PR DESCRIPTION
**Pull request by AD9018**
## Description

This fix addresses the Judge Table but specifically the ability for a user to be able to select multiple search results for any category. 

Co-Authored-By: Ricky Klusmeier

Fixes # (issue)
The user can now select multiple inputs offering the user a better experience and more capability. 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by joshuascan — CHANGES_REQUESTED** [review]
Great work on this! The main thing that needs to be addressed though is that some of the variable names in CaseTable.js (see my comments in the file) have been changed back to their old variable names, which would very likely break some stuff. There are also a couple console logs that should be removed. Other than that though, everything looks good and it seems like you guys have gotten quite comfortable with ANT design!

File: src/components/pages/Cases/CaseTable.js [hunk 1]
```diff
@@ -12,33 +12,25 @@ import {
...
 import './CaseTable.less';
 import CaseDetails from '../CaseOverview/CaseDetails';
-//
-// Case column utils import
-import { case_columns } from '../../../utils/case_utils/case_columns';
+//* Case column utils import
+import { case_columns } from '../../../utils/case_utils/case_columns.js';
 
-// Utils for filter keywords
+//* Utils for filter keywords
```
[the remark this answers is not in the document]
Comment (paulstgermain) [reply src/components/pages/Cases/CaseTable.js:21]: The asterisks come from the Better Comments extension for VS Code - with that installed, adding symbols to the starts of your comments apply different colors, and it applies the asterisks automatically.

[the remark this answers is not in the document]
Comment (joshuascan) [reply src/components/pages/Cases/CaseTable.js:21]: Good to know! That definitely sounds useful.

**Review by ctscofield — APPROVED** [review]
A little difficult for me to see what all the changes meant throughout the commits but I do see a lot of changes that made readability a ton easier as well as the code portion that added the feature and the bug fixing itself. Pretty big fan of your progress so far, keep it up!

**Review by willmoon19 — APPROVED** [review]
A really good feature to add! I think you did a great job working on this with the other developers. Since josh pointed out the errors and already made a request for change, once those go through I think it'll be perfect.

**Review by prodhc — APPROVED** [review]
Great work Adam and Ricky, It all looks great besides the issues that were already pointed out.

**Comment by paulstgermain** [comment]
test comment

File: src/components/pages/Cases/CaseTable.js [hunk 2]
```diff
@@ -80,10 +69,10 @@ export default function CaseTable(props) {
     outcome: '',
     country_of_origin: '',
     gender: '',
-    type_of_persecution: '',
+    type_of_violence: '',
     indigenous_group: '',
     applicant_language: '',
-    credibility: '',
+    credible: '',
```
[the remark this answers is not in the document]
Comment (RickyKlusmeier) [reply src/components/pages/Cases/CaseTable.js:75]: These will be fixed in the final PR in the clean up branch

**Review by RickyKlusmeier — APPROVED** [review]

File: src/components/pages/Cases/CaseTable.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (joshuascan) [reply src/components/pages/Cases/CaseTable.js:75]: Sounds good. Will change to approved.

**Review by joshuascan — APPROVED** [review]

**Review by RickyKlusmeier — APPROVED** [review]

**Review by matthewblackbu — APPROVED** [review]
Looks good to me I can't find anything that wasn't pointed out already good work Ricky and Adam

**Review by leecrowe — APPROVED** [review]
It looks like plenty of necessary changes were made to make this more readable and functional. Awesome work you guys!

**Pull request merged**